src/assets/errors.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
assets folder is for static assets. Errors json should go to some module configuration